src/components/BasicMap/BasicMap.js

```diff
@@ -132,11 +136,34 @@ class BasicMap extends Component {
       zoom,
     } = this.props;
 
+    this.dfltInteractions = defaultInteractions({
+      altShiftDragRotate: false,
+      pinchRotate: false,
+    });
+
+    if (map) {
+      if (interactions) {
+        interactions.forEach(i => {
+          map.addInteraction(i);
+        });
+      } else {
+        // Default interaction without pinch rotate and drag rotate.
+        map
+          .getInteractions()
+          .getArray()
+          .forEach(i => {
+            if (i instanceof DragRotate || i instanceof PinchRotate) {
+              map.removeInteraction(i);
+            }
+          });
+      }
+    }
```

Comment:
I don't think you need this block . If user provide a Map with interactions it's up to him to manage his interactions.

Reply:
My requirement for this task is to deactivate rotation,pinch rotate, etc by default in react-spatial.

Reply:
This requirement will not be fulfilled, if I don;t unset by default rotation. But still allow the user to add interactions directly in props if needed.

Reply:
quote from mario: 'Wenn keine Map übergeben wird, initialisieren wir die Karte ohne Rotation.'
so I removed it :+1: